docs/developer-center/guides/04-Quickstart.md

```diff
@@ -935,6 +927,30 @@ As we can see, there are clearly 3 stores that have lower revenue per person. Th
 
 To learn more about discovering the data you want, check out the [data discovery guide](/developers/cartoframes/guides/Data-discovery). To learn more about enriching your data check out the [data enrichment guide](/developers/cartoframes/guides/Data-enrichment/).
 
+### Save your data
+
+The `CartoDataFrame` structure allows you to save your data directly in your CARTO account, so you can use it in different notebooks directly. Once it's saved, you can download it whenever you want, but also use it in your visualizations using the `table_name` you've given to the dataset. For example, let's save the `stores_cdf` under the name of `starbucks_stores_analysis`:
+
+```python
+stores_cdf.to_carto('starbucks_stores_analysis')
+```
+
+Now, we can visualize the table directly from CARTO as follows:
+
+```python
+# Using the CartoDataFrame:
+# Map(size_continuous_layer(stores_cdf, 'rev_pop', 'Revenue per person ($)'))
```

Comment:
I like having this here but also find it confuses the workflow a little. If we're going to visualize the data, how about visualizing first with the table and they say we could directly visualize from the dataframe below this?

Reply:
In my opinion, I think it's better to avoid adding more steps to generate the visualization as the first option, maybe it's better to remove the comments cause they might be confusing.

Reply:
Removing sounds good but makes me feel like we're missing a good opportunity to highlight the amazing feature of `Layer(dataframe)`

Reply:
Oh! but then that's a different thing, right? Maybe didn't understand it very well. Do you mean removing the `Map` class when creating the visualization?

Reply:
I agree with @andy-esch first comment

Reply:
done! ✨